corehq/apps/hqwebapp/static/hqwebapp/js/components/inline_edit.js

```diff
@@ -144,4 +144,17 @@ hqDefine('hqwebapp/js/components/inline_edit', [
         },
         template: '<div data-bind="template: { name: \'ko-inline-edit-template\' }"></div>',
     };
+
+    ko.components.register('inline-edit', component);
+
+    $(function () {
+        _.each($('inline-edit'), function (el) {
+            var $el = $(el);
+            if ($el.data('apply-bindings') !== false) {
+                $(el).koApplyBindings();
+            }
+        });
```

Comment:
I notice this block in each component, as it used to be in _components.ko.js_. Do you think it makes sense to create a common registration function? Specifically, the `$el.data('apply-bindings')` logic isn't that obvious, so I worry about the duplication.

Reply:
yeah, I don't feel great about this duplication either, I'll take a look at cleaning it up.

Reply:
Cleaned up in 61776dbfb0d